src/models/decisions/decision.ts

```diff
@@ -0,0 +1,17 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+
+export enum VerificationDecision {
+    FALSE = "false",
+    TRUE = "true",
+    UNSURE = "unsure",
+    SKIP = "skip",
+}
+
+// A polymorphic structure that can be used to represent any decision type
+// that implements the interface
+export interface IDecision {
+    confirmed: EnumValue<VerificationDecision>;
+    identifier: symbol;
+    type: string;
+    colorIndex: number;
```

Comment:
Why does this model know anything about colors?

Suggestion: store a decision identifier / key. Could be a type alias for number: `type DecisionId = number`. Or could be a decision source (I believe this what you intended `origin` to mean in the previous patch).